Sources/PostgreSQL/Connection.swift

```diff
@@ -22,17 +22,11 @@ public final class Connection: ConnInfoInitializable {
         let string: String
         
         switch conninfo {
-        case .raw(let ci):
-            string = ci
+        case .raw(let info):
+            string = info
         case .params(let params):
-            var ci = ""
-            
-            params.forEach { (key, value) in
-                ci += "\(key)='\(value)'"
-            }
-        
-            string = ci
-        case .basic(host: let host, port: let port, database: let database, user: let user, password: let password):
+            string = params.map({ (key, value) in "\(key)='\(value)'" }).reduce("", +)
```

Comment:
Both look good to me, I can change it if thats preferred. Would it be too much to write it like this: `params.map({ "\($0)='\($1)'" }).joined()` ?